http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/leoPubsubMessageSubscriberModels.scala

```diff
@@ -569,10 +570,11 @@ object LeoPubsubCodec {
     Decoder.forProduct2("name", "resourceId")(StorageContainerResponse.apply)
 
   implicit val createAppV2Decoder: Decoder[CreateAppV2Message] =
-    Decoder.forProduct7("appId",
+    Decoder.forProduct8("appId",
                         "appName",
                         "workspaceId",
                         "cloudContext",
+                        "userAccessToken",
```

Comment:
Just mentioning: another way to do this besides passing the original user-token in the PubSub message would be to mint a user-pet token in `AKSInterpreter` using `samDao.getCachedArbitraryPetAccessToken`. We are already using that method in `AKSInterpreter` for other things:

https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala#L367

I guess benefits of that would be (a) no need to pass a sensitive token through PubSub; and (b) the minted token would always have a lifetime of 60 minutes; while the user-token-refresh is handled by the UI so might have a remaining lifetime of <60 mins.